sci-biology/velvet/velvet-1.2.10.ebuild

```diff
@@ -26,11 +26,13 @@ src_prepare() {
 	if ! use doc; then
 		sed -i -e '/default :/ s/doc//' "${S}"/Makefile || die
 	fi
-	elog "Upstream recommendes using -O3 in CFLAGS"
+	elog "Upstream recommends using -O3 in CFLAGS"
 	echo
-	elog "To adjust the MAXKMERLENGTH or CATEGORIES parameters as described in the manual,"
-	elog "please set the variables VELVET_MAXKMERLENGTH or VELVET_CATEGORIES in your"
-	elog "environment or /etc/make.conf, then re-emerge the package. For example:"
+	elog "To adjust the MAXKMERLENGTH, CATEGORIES, BIGASSEMBLY, LONGSEQUENCES parameters"
+	elog "as described in the PDF manual, please set the variables by prepending VELVET_ in"
+	elog "front of it. For example VELVET_MAXKMERLENGTH, VELVET_CATEGORIES, ..."
+	elog "Set them either in your environment or in /etc/portage/make.conf, then re-emerge"
+	elog "the package. For example:"
 	elog "  VELVET_MAXKMERLENGTH=NN emerge [options] velvet"
 	MAKEOPTS+=" -j1"
```

Comment:
This is bad (but common style in Gentoo): you shouldn't ever muck with `MAKEOPTS`, instead appending `-j1` to `emake` calls.